source/test/F0.Minesweeper.Components.Tests/CellTests.cs

```diff
@@ -148,5 +187,72 @@ public void OnRightClick_StatusManagerReturnsUndefined_ChangesTextToSpecialUndef
 			// Assert
 			componentUnderTest.MarkupMatches(expectedMarkup);
 		}
+
+		[Theory]
+		[InlineData(true, CellStatusType.Mine, 3, '☢')]
```

Comment:
We found that we will bring new themes in the future - but would definitely like to see the "classic"/"retro"/"1.0.0" theme as an option to remain (or maybe come back later 😉) - so we'll leave this as is because it will change either way when supporting themes.